command/crypto/jwt/sign.go

```diff
@@ -57,6 +59,8 @@ A JWT signed using JWS has three parts:
 
 For examples, see **step help crypto jwt**.`,
 		Flags: []cli.Flag{
+			flags.X5cCert,
+			flags.X5cKey,
```

Comment:
Move them down.

Reply:
Reuse `--key`? Whatever looks better.

Reply:
optically I think it's nice to have `x5c-key` available, but as you said I can add it to the definition of the `key` flag.